src/components/YearGraph/YearGraph.tsx

```diff
@@ -0,0 +1,159 @@
+// FourthContainer Imported
+import { FourthContainer } from "./YearGraphStyled";
+import { StockChartPlaceholder } from "./YearGraphStyled";
+import { PlaceholderMessage } from "./YearGraphStyled";
+// FifthContainer Imported
+import { FifthContainer } from "./YearGraphStyled";
+import { BlockOne } from "./YearGraphStyled";
+import { BlockOneContainerOne } from './YearGraphStyled';
+import { BlockOneContainerTwo } from './YearGraphStyled';
+import { BlockOneContainerThree } from './YearGraphStyled';
+import { BlockOneLabelOne } from './YearGraphStyled';
+import { BlockOneLabelTwo } from './YearGraphStyled';
+import { BlockOneLabelThree } from './YearGraphStyled';
+import { BlockOneLabelFour } from './YearGraphStyled';
+import { BlockOneLabelFive } from './YearGraphStyled';
+import { BlockOneLabelSix } from './YearGraphStyled';
+import { BlockTwo } from "./YearGraphStyled";
+import { BlockTwoContainerOne } from './YearGraphStyled';
+import { BlockTwoContainerTwo } from './YearGraphStyled';
+import { BlockTwoContainerThree } from './YearGraphStyled';
+import { BlockTwoLabelOne } from './YearGraphStyled';
+import { BlockTwoLabelTwo } from './YearGraphStyled';
+import { BlockTwoLabelThree } from './YearGraphStyled';
+import { BlockTwoLabelFour } from './YearGraphStyled';
+import { BlockTwoLabelFive } from './YearGraphStyled';
+import { BlockTwoLabelSix } from './YearGraphStyled';
+import { BlockThree } from "./YearGraphStyled";
+import { BlockThreeContainerOne } from './YearGraphStyled';
+import { BlockThreeContainerTwo } from './YearGraphStyled';
+import { BlockThreeContainerThree } from './YearGraphStyled';
+import { BlockThreeLabelOne } from './YearGraphStyled';
+import { BlockThreeLabelTwo } from './YearGraphStyled';
+import { BlockThreeLabelThree } from './YearGraphStyled';
+import { BlockThreeLabelFour } from './YearGraphStyled';
+import { BlockThreeLabelFive } from './YearGraphStyled';
+import { BlockThreeLabelSix } from './YearGraphStyled';
+// SixthContainer Imported
+import { SixthContainer } from "./YearGraphStyled";
+import { TitleText } from "./YearGraphStyled";
+import { Subtext } from "./YearGraphStyled";
+import { ButtonSeven } from "./YearGraphStyled";
+import { StyledLink } from './YearGraphStyled';
+import { StyledLinkTwo } from './YearGraphStyled';
+
+// Creating a YearGraph JSX Element 
+const YearGraph = ( ): JSX.Element => {
+  return (
+      <Wrapper>
+        <MainContainer>
+          <LabelOne>19.00</LabelOne>
+          <LabelTwo> <FontAwesomeIcon icon={faCaretUp}/> </LabelTwo>
+          <LabelThree>0.10(0.53%)</LabelThree>
+        </MainContainer>
+
+        <SecondContainer>
+        <LabelFour>19.33</LabelFour>
+          <LabelFive> <FontAwesomeIcon icon={faCaretUp}/> </LabelFive>
+          <LabelSix>0.33(1.74%)</LabelSix>
+          <LabelSeven>After Hours . June 29, 7:59PM EDT . Market Closed </LabelSeven>
+        </SecondContainer>
+
+        <ThirdContainer>
+          <ButtonOne>Day</ButtonOne>
```

Comment:
Maybe we can refactor stuff like this? React is useful for recreating similar elements -- just add props